app/services/creators/capital_eligibility_creator.rb

```diff
@@ -18,22 +18,35 @@ def call
     def create_eligibility(ptc)
       return if eligibility_record_exists?(ptc)
 
-      @summary.eligibilities.create!(
-        proceeding_type_code: ptc,
-        upper_threshold: upper_threshold(ptc),
-        lower_threshold:,
-        assessment_result: "pending",
-      )
+      if @assessment.level_of_representation == "controlled"
+        @summary.eligibilities.create!(
+          proceeding_type_code: ptc,
+          upper_threshold: controlled_threshold(ptc),
+          lower_threshold: controlled_threshold(ptc),
+          assessment_result: "pending",
+        )
+      else
+        @summary.eligibilities.create!(
+          proceeding_type_code: ptc,
+          upper_threshold: upper_threshold(ptc),
+          lower_threshold:,
+          assessment_result: "pending",
+        )
+      end
     end
 
-    def lower_threshold
-      if @assessment.level_of_representation == "controlled"
-        Threshold.value_for(:capital_lower_controlled, at: @assessment.submission_date)
+    def controlled_threshold(ptc)
+      if ptc.to_sym.in? CFEConstants::IMMIGRATION_AND_ASYLUM_PROCEEDING_TYPE_CCMS_CODES
+        Threshold.value_for(:capital_first_tier_tribunal_controlled, at: @assessment.submission_date)
       else
-        Threshold.value_for(:capital_lower_certificated, at: @assessment.submission_date)
+        @assessment.proceeding_types.find_by!(ccms_code: ptc).capital_upper_threshold
```

Comment:
We might as well use the `upper_threshold(ptc)` one-liner here since we have it.